src/app/content/highlights/components/Card.tsx

```diff
@@ -109,189 +133,41 @@ const Card = (props: Props) => {
   };
 
   const commonProps = {
-    className: props.className,
-    isFocused: props.isFocused,
-    onBlur: props.blur,
+    className,
+    isFocused,
+    onBlur: blur,
```

Comment:
since an onBlur function that does something totally different exists, you might want to do some more refactoring to make this name consistent